GliaWidgets/Public/Environment.swift

```diff
@@ -4,10 +4,13 @@ import Foundation
 public enum Environment {
     /// Europe
     case europe
+
     /// USA
     case usa
+
     /// Beta environment. For development use.
     case beta
+    
```

Comment:
Please fix this SwiftLint issue

Reply:
fixed [db0bd5d](https://github.com/salemove/ios-sdk-widgets/pull/887/commits/db0bd5dd0c0d19c607cfb3ccefc10575b54df171)